### PR TITLE
sendNotifications + updateNotificationStatus 

### PR DESCRIPTION
**Pull request by bergieboy**

### CODE REVIEW

File: src/main/kotlin/com/openlattice/chronicle/notifications/NotificationStatus.kt [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+/**
+ *  @author Todd Bergman <todd@openlattice.com>
+ *  https://support.twilio.com/hc/en-us/articles/223134347-What-are-the-Possible-SMS-and-MMS-Message-Statuses-and-What-do-They-Mean-
+ *  Finalized Message Delivery Status
+ *  After a message has been sent, the following final delivery status options are possible:
+ */
+
+enum class NotificationStatus {
+    delivered,
```
Comment (geekbeast) [inline src/main/kotlin/com/openlattice/chronicle/notifications/NotificationStatus.kt:11]: Should be caps.

Comment (geekbeast) [reply src/main/kotlin/com/openlattice/chronicle/notifications/NotificationStatus.kt:11]: or PascalCase

File: src/main/kotlin/com/openlattice/chronicle/notifications/Notification.kt [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    val candidateId: UUID,
+    val organizationId: UUID,
+    val studyId: UUID,
+    val createdAt: OffsetDateTime = OffsetDateTime.now(),
+    var updatedAt: OffsetDateTime = OffsetDateTime.now(),
+    var status: String,
+    var messageId: String,
+    val type: String,
+    val body: String,
+    val email: String?,
```
Comment (geekbeast) [inline src/main/kotlin/com/openlattice/chronicle/notifications/Notification.kt:21]: default to null (matches deserialization behavior) to make it easier to construct objects in kotlin

**Review by geekbeast — APPROVED** [review]
Minor feedback.

File: src/main/kotlin/com/openlattice/chronicle/notifications/NotificationStatus.kt (hunk 1 above)
Comment (UnsungHero97) [reply src/main/kotlin/com/openlattice/chronicle/notifications/NotificationStatus.kt:11]: This enum is meant to represent Twilio constants:

https://support.twilio.com/hc/en-us/articles/223134347-What-are-the-Possible-SMS-and-MMS-Message-Statuses-and-What-do-They-Mean-

File: src/main/kotlin/com/openlattice/chronicle/notifications/NotificationStatus.kt [hunk 3]
```diff
@@ -0,0 +1,16 @@
+package com.openlattice.chronicle.notifications
+
+/**
+ *  @author Todd Bergman <todd@openlattice.com>
+ *  https://support.twilio.com/hc/en-us/articles/223134347-What-are-the-Possible-SMS-and-MMS-Message-Statuses-and-What-do-They-Mean-
+ *  Finalized Message Delivery Status
+ *  After a message has been sent, the following final delivery status options are possible:
+ */
+
+enum class NotificationStatus {
```
Comment (UnsungHero97) [inline src/main/kotlin/com/openlattice/chronicle/notifications/NotificationStatus.kt:10]: This is a duplicate of `MessageStatus`. Do you need both?

https://github.com/openlattice/chronicle-api/blob/develop/src/main/java/com/openlattice/chronicle/data/MessageStatus.kt

**Review by UnsungHero97 — CHANGES_REQUESTED** [review]

File: src/main/kotlin/com/openlattice/chronicle/notifications/NotificationStatus.kt (hunk 1 above)
Comment (geekbeast) [reply src/main/kotlin/com/openlattice/chronicle/notifications/NotificationStatus.kt:11]: Got it. Makes sense.

File: src/main/kotlin/com/openlattice/chronicle/notifications/NotificationStatus.kt (hunk 3 above)
Comment (bergieboy) [reply src/main/kotlin/com/openlattice/chronicle/notifications/NotificationStatus.kt:10]: MessageStatus is not in v3 branch

**Pull request merged**